devel/util-linux/Portfile

```diff
@@ -25,6 +25,16 @@ checksums           rmd160  e07164962fb506f0ceb1e2c5057c918e2972bb91 \
                     sha256  f261b9d73c35bfeeea04d26941ac47ee1df937bd3b0583e748217c1ea423658a \
                     size    4663072
 
+if { ${os.platform} eq "darwin" && ${os.major} < 14 } {
+    # versions after this use faccessat and are compatible with darwin 14 and newer
```

Comment:
It would be good if you could add the URL of the upstream bug report about this.

Reply:
entered upstream: <https://github.com/karelzak/util-linux/issues/716>

Reply:
will add in to the portfile comments